packages/backend/src/scripts/create-migration.ts

```diff
@@ -0,0 +1,94 @@
+import execa from 'execa';
+import * as process from 'process';
+import * as readline from 'readline';
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
+
+const migrationName = process.argv[2];
+
+if (!migrationName) {
+    console.error(
+        'Error: Migration name is required, e.g create-migration add_user_role',
+    );
+    console.error('Usage: pnpm create-migration <migration-name>');
+    process.exit(1);
+}
+
+// Prompt user to choose between OSS and EE migrations
+const promptUser = (): Promise<'ee' | 'oss'> =>
+    new Promise<'ee' | 'oss'>((resolve) => {
+        rl.question(
+            'Create migration in (1) OSS or (2) EE? [1/2]: ',
+            (answer: string) => {
+                rl.close();
+
+                const normalizedAnswer = answer.trim().toLowerCase();
+
+                if (normalizedAnswer === '1' || normalizedAnswer === 'oss') {
+                    resolve('oss');
+                } else if (
+                    normalizedAnswer === '2' ||
+                    normalizedAnswer === 'ee'
+                ) {
+                    resolve('ee');
+                } else {
+                    console.log('Invalid choice. Defaulting to OSS.');
+                    resolve('oss');
+                }
+            },
+        );
+    });
+
+const createMigration = async (): Promise<void> => {
+    const migrationType = await promptUser();
+
+    try {
+        // Define the migration directory based on user choice
+        let migrationDir: string;
+
+        if (migrationType === 'ee') {
+            // Check if license key exists and is valid
+            const licenseKey = process.env.LIGHTDASH_LICENSE_KEY;
+            if (!licenseKey || licenseKey.trim() === '') {
+                console.error(
+                    'Error: LIGHTDASH_LICENSE_KEY is required for EE migrations.',
+                );
+                console.error(
+                    'Please set the environment variable and try again.',
+                );
+                process.exit(1);
+            }
+
+            migrationDir = 'ee/database/migrations';
+            console.log('Creating EE migration...');
+        } else {
+            migrationDir = 'database/migrations';
+            console.log('Creating OSS migration...');
+        }
+
+        // Run migration
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call
+        const { stdout } = await execa('knex', [
+            'migrate:make',
+            migrationName,
+            '--knexfile',
+            'src/knexfile.ts',
+            '--migrations-directory',
+            `src/${migrationDir}`,
```

Comment:
Something is wrong with the paths. 

Trying inside docker:
<img width="840" alt="Screenshot 2025-02-27 at 16 45 15" src="https://github.com/user-attachments/assets/806aa637-265b-443e-b846-cc4fb56f3aff" />

Trying outside docker:
<img width="1551" alt="Screenshot 2025-02-27 at 16 51 15" src="https://github.com/user-attachments/assets/ae996d75-a418-4633-9322-9d660d63e7b4" />

Trying inside docker, and running command inside backend folder:
<img width="819" alt="Screenshot 2025-02-27 at 16 52 02" src="https://github.com/user-attachments/assets/aa168085-bfe0-4722-91f3-fc8fdc74faf7" />

Reply:
🤦 thank you, fixed! 
<img width="584" alt="image" src="https://github.com/user-attachments/assets/156c400e-e631-4c4b-9bdf-f8c59d5d7329" />